core/src/main/java/org/infinispan/manager/EmbeddedCacheManager.java

```diff
@@ -319,5 +320,11 @@ default ClusterExecutor executor() {
       throw new UnsupportedOperationException();
    }
 
-
+   /**
+    * Returns an entry point for a Health Check API.
+    *
+    * @since 9.0
+    * @return Health API for this {@link EmbeddedCacheManager}.
```

Comment:
I see we have a task to add `@return` to javadocs in documentation mini-sprint. Having this in mind I would like to leave it as is.